framework/components/ecl-breadcrumbs/ecl-breadcrumbs.config.js

```diff
@@ -23,16 +23,4 @@ module.exports = {
       },
```

Comment:
I'd remove the `isInternal: false` lines to make it simpler (and just use `isInternal is defined and is isInternal == true` in the template)